contracts/UFragmentsPolicy.sol

```diff
@@ -249,7 +249,7 @@ contract UFragmentsPolicy is Ownable {
      * @return Computes the total supply adjustment in response to the exchange rate
      *         and the targetRate.
      */
-    function computeSupplyDelta(uint256 rate, uint256 targetRate) private view returns (int256) {
+    function computeSupplyDelta(uint256 rate, uint256 targetRate) internal view returns (int256) {
```

Comment:
Confusingly, I think `private` -> `internal` has both a gas change (jump w/ no memory clear) and a functional change (visibility within derived contracts). Just pointing out the change to visibility, but I think that's ok.

https://docs.soliditylang.org/en/v0.8.0/control-structures.html?highlight=internal#internal-function-calls
https://docs.soliditylang.org/en/v0.8.0/contracts.html?highlight=private#visibility-and-getters

Reply:
Noted. Yea. That's weird